.github/workflows/jdk-outreach-workflow.yml

```diff
@@ -117,53 +110,14 @@ jobs:
       with:
         java-version: ${{ matrix.java }}
     - name: Maven Build
-      run: mvn clean install -B -pl debezium-connector-mongodb -am -Passembly -Dformat.formatter.goal=validate
-        -Dformat.imports.goal=check -Dversion.mongo.server=3.2 -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
-  mongodb-36:
```

Comment:
So after discussing with @jpechane, I realize now this actually should remain as-is. I was under the impression that the MongoDB version matrix testing was related to the Outreach job, but turns out it isn't. So actually for MongoDB things are already as they ideally should be for all connectors/jobs: test all the required versions on GH Actions (provided its capacity and limits are enough for us to do so).

Reply:
Alright. Will add them back.